hibernate-core/src/test/java/org/hibernate/test/legacy/MultiTableTest.java

```diff
@@ -249,6 +252,7 @@ public void testConstraints() throws Exception {
 	}
 
 	@Test
+	@RequiresDialectFeature(DialectChecks.SupportNoWait.class)
```

Comment:
can this be changed with `@SkipForDialect( value = CockroachDB192Dialect.class`

Reply:
done